README.md

```diff
@@ -77,38 +77,59 @@ Adjusted application properties have to be provided. An application properties t
 version=@project.version@
 # Enviroment where application will be running
 environment=Local
+# Maximum number of workflows that can be triggered in bulk (Default value 10)
+application.maximumNumberOfWorkflowsInBulkRun=10
+```
+```
+# Unique app id, for easier application identification
+appUniqueId=
+```
+```
+# Health check settings
+health.databaseConnection.timeoutMillis=120000
+health.yarnConnection.testEndpoint=/cluster/cluster
+health.yarnConnection.timeoutMillis=60000
 ```
 ```
 # How will users authenticate. Available options: inmemory, ldap
 auth.mechanism=inmemory
+#If set, all users that do not have admim role will not have access to admin protected endpoints
+auth.admin.role=
 # INMEMORY authentication: username and password defined here will be used for authentication.
 auth.inmemory.user=user
 auth.inmemory.password=password
+auth.inmemory.admin.user=admin-user
+auth.inmemory.admin.password=admin-passowrd
```

Comment:
The default value is `hyperdriver-admin-user` and `hyperdriver-admin-password`. I'd change it to that, and also in the `application.properties`

Reply:
Done